CONTRIBUTING.md

```diff
@@ -1,27 +1,39 @@
 # How to contribute
-We ❤️ pull requests. If you'd like to fix a bug, contribute a feature or
-just correct a typo, please feel free to do so, as long as you follow
-our [Code of Conduct](https://github.com/Shopify/slate/blob/master/CODE_OF_CONDUCT.md).
+We ❤️ pull requests. If you'd like to fix a bug, contribute a feature or just correct a typo, please feel free to do so, as long as you follow our [Code of Conduct](https://github.com/Shopify/slate/blob/master/CODE_OF_CONDUCT.md).
 
-The Slate theme scaffolding is [intentionally barebones](https://shopify.github.io/slate/theme/#intentionally-blank)
-to provide an unopinionated starting point.  If you're thinking of adding a new
-feature, consider opening an issue first to discuss it to ensure it aligns to
-the direction of the project (and potentially save yourself some time!).
+If you're thinking of adding a big new feature, consider opening an issue first to discuss it to ensure it aligns to the direction of the project (and potentially save yourself some time!).
+
+This repo is a [monorepo](https://github.com/babel/babel/blob/master/doc/design/monorepo.md) consisting of multiple packages and is managed using [Lerna](https://github.com/lerna/lerna).
+
+The `slate-theme` scaffolding is [intentionally barebones](https://shopify.github.io/slate/theme/#intentionally-blank) to provide an unopinionated starting point.  If you're thinking of adding a new feature, consider opening an issue first to discuss it to ensure it aligns to the direction of the project (and potentially save yourself some time!).
```

Comment:
The **"If you're thinking of adding...** sentence is duplicated - once here and once above.